storybook/stories/metric/1_basic.story.tsx

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { select, text, color, number } from '@storybook/addon-knobs';
+import React from 'react';
+
+import { Chart, Metric, Settings } from '@elastic/charts';
+import { KIBANA_METRICS } from '@elastic/charts/src/utils/data_samples/test_dataset_kibana';
+
+import { useBaseTheme } from '../../use_base_theme';
+
+export const Example = () => {
+  const title = text('title', '21d7f8b7-92ea-41a0-8c03-0db0ec7e11b9');
+  const subtitle = text('subtitle', 'Cluster CPU usage');
+  const progressOrTrend = select(
+    'progress or trend',
+    {
+      trend: 'trend',
+      bar: 'bar',
+      none: 'none',
+    },
+    'trend',
+  );
+  const progressBarDirection = select(
+    'progress bar direction',
+    { horizontal: 'horizontal', vertical: 'vertical' },
+    'vertical',
+  );
+  const maxDataPoints = number('trend data points', 30, { min: 0, max: 50, step: 1 });
+  const trendShape = select(
+    'trend shape',
+    {
+      area: 'area',
+      bar: 'bars',
```

Comment:
This type is not being enforced

```suggestion
      bar: 'bar',
```